app/assets/javascripts/calendar.es6

```diff
@@ -11,6 +11,21 @@ class Calendar {
     const defaultConfig = {
       allDaySlot: false,
       cookieName: this.$el.attr('id') || 'calendar',
+      customButtons: {
+        jumpToDate: {
+          text: 'Jump to date',
+          click: (event) => {
+            this.$jumpToDateEl.val(this.getCurrentDate());
+
+            const dateRangePicker = this.$jumpToDateEl.data('daterangepicker');
+
+            dateRangePicker.setStartDate(this.getCurrentDate());
+            dateRangePicker.setEndDate(this.getCurrentDate());
```

Comment:
Worth declaring `this.getCurrentDate()` as a variable as we're referencing it 3 times?